pkg/aws/cloud.go

```diff
@@ -28,8 +28,9 @@ type Cloud interface {
 }
 
 // NewCloud constructs new Cloud implementation.
-func NewCloud(cfg CloudConfig, metricsRegisterer prometheus.Registerer) (Cloud, error) {
+func NewCloud(cfg CloudConfig, metricsRegisterer prometheus.Registerer, UseAwsDualStackEndpoint int, UseAwsFIPSEndpoint int) (Cloud, error) {
 	sess := session.Must(session.NewSession(aws.NewConfig()))
+	sessAPPMESH := session.Must(session.NewSession(aws.NewConfig()))
```

Comment:
nit: can you change this to `sessAppMesh`

Reply:
Also, can you add a comment explaining that due to the unavailability of dual stack endpoints (DNS ending in api.aws) that we have to have separate configs for App Mesh vs others